docs/en/05_Contributing/01_Code.md

```diff
@@ -112,16 +112,18 @@ The current GitHub labels are grouped into five sections:
 | change/major | A change for the next major release (eg: 4.0) |
 | change/minor | A change for the next minor release (eg: 3.x) |
 | change/patch | A change for the next patch release (eg: 3.1.x) |
-| impact/critical | Broken functionality without workarounds, affecting major usage flows. Should target all [supported minor releases](release_process#supported-versions) |
-| impact/high | Broken functionality with workarounds available, affecting major usage flows |
+| impact/critical | Broken functionality/experience without any workarounds, or an enhancement that is required to enable a critical task. Typically affecting major usage flows or core interactions. Should target all [supported minor releases](release_process#supported-versions) |
```

Comment:
The last part of this sentence isn't relevant for enhancements, only for bugs, since we can't put enhancements into minor release branches